models/applicationSubmissionsModel.js

```diff
@@ -0,0 +1,60 @@
+const db = require('../database/dbConfig');
+
+async function findById(id) {
+  return db('application_submissions')
+    .where({ id })
+    .first();
+}
+
+async function findAllByCampaignId(campaign_id) {
+  return db('skilled_impact_requests')
+    .where({ campaign_id })
+    .join(
+      'application_submissions',
+      'skilled_impact_requests.id',
+      'application_submissions.skilled_impact_request_id'
+    )
+    .select(
+      'skilled_impact_requests.campaign_id',
+      'application_submissions.*'
+    );
+}
+
+async function insert(submission) {
+  return db('application_submissions')
+    .insert(submission, ['*']);
```

Comment:
nit: can you use the [`returning`](http://knexjs.org/#Builder-returning) function instead of passing in the asterisk to `insert`? I think it makes the code a bit more readily understandable.